components/brave_wallet/common/meld_integration.mojom

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+module brave_wallet.mojom;
+
+import "mojo/public/mojom/base/time.mojom";
+
+struct MeldLogoImages {
```

Comment:
I wonder do we really need all the things in C++?
Do we interact the structure somehow except parsing and passing to WebUI?

Any option to move parsing part to TypeScript?

Reply:
moved response parsing into separate thread

Reply:
The current convention is to do all the parsing in C++, to simplify integration on desktop, android, and ios.